packages/create-sitecore-jss/src/templates/nextjs/package.json

```diff
@@ -32,7 +32,7 @@
     "@sitecore-jss/sitecore-jss-nextjs": "^20.0.0-canary",
     "graphql": "~15.4.0",
     "graphql-tag": "^2.11.0",
-    "next": "^12.0.4",
+    "next": "12.0.4",
```

Comment:
Why it was changed to the `fixed` version? We should have latest

Reply:
@ambrauer do you know if there was a reason for this?
This change was there when I picked up the work.

Reply:
@sc-addypathania None that I know of. I'd assume that was unintentional.